.github/workflows/lint.yml

```
@@ -23,6 +23,7 @@ jobs:
       - run: yarn install --immutable --immutable-cache --check-cache
       - uses: hashicorp/setup-terraform@v3
       - uses: terraform-linters/setup-tflint@v4
+      - run: tflint --init --config=terraform/.tflint.hcl
```

```suggestion
      - run: tflint --chdir=terraform --init
```

I think this is slightly nicer as it's consistent with the commands in `hk.pkl` although I think it's functionally the same.

---

Done